app/views/streams/main_stream.html.haml

```diff
@@ -4,8 +4,12 @@
 
 - content_for :head do
   - if AppConfig.chat.enabled?
-    = javascript_include_tag :jsxc, id: 'jsxc',
-      data: { endpoint: get_bosh_endpoint }
+    - if AppConfig.chat.frontend.eql? "converse"
+      = javascript_include_tag :converse, id: 'converse',
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Space inside { detected.
- Space inside } detected.